pkg/analysis_server/test/services/correction/fix_test.dart

```diff
@@ -6074,6 +6074,109 @@ class A {
 ''');
   }
 
+  test_replaceWithConditionalAssignment_withCodeBeforeAndAfter() async {
+    String src = '''
+class Person {
+  String _fullName;
+  void foo() {
+    print('hi);
```

Comment:
The string literal isn't closed correctly. It must be the case that the parser recovers at the end of the line and the extra errors don't cause issues for the test framework. Still, it would be good to fix it (here and below).